backend/cmd/cmd.go

```diff
@@ -101,7 +101,7 @@ func Serve(cmd *cobra.Command) error {
 		openDefaultBrowser(c.Config().IsSecure, c.Config().ListenAddr)
 	}
 
-	if !isSecure && !strings.Contains(c.Config().ListenAddr, "localhost") {
+	if !isSecure && !strings.Contains(c.Config().ListenAddr, "[::]:7080") && !strings.Contains(c.Config().ListenAddr, "localhost") {
```

Comment:
Wouldn't `127.0.0.1` be a saner default than `localhost`? I've seen some setups without the `localhost` name being mapped to the loopback interface.

Reply:
Thank you for this clear explanation. Looks good to me!